docs/config/README.md

```diff
@@ -301,14 +303,63 @@ $shell\
 $character"""
 ```
 
-If you just want to extend the default format, you can use `$all`;
-modules you explicitly add to the format will not be duplicated. Eg.
+If you want to extend the default format, you can use `$all`;
+modules you explicitly add to the format will not be duplicated, but replace earlier declarations.
+
+For example, to move the directory to the end of the prompt (also moving it past `$line_break` onto the second line):
 
 ```toml
-# Move the directory to the second line
+# Move the directory to the end of the second line
 format = "$all$directory$character"
 ```
 
+If you use only a subset of the modules you may prefer to use a much shorter format setting.
+
+Modules that are not referred in `format` will not query for status information. Reducing the number of modules in `format` (removing unused ones) leads to a performance improvement.
+
+Setting `format` to an empty string will lead to a completely empty prompt.
+
+```toml
+format = ''
+```
+
+To show only the directory and prompt character:
+
+```toml
+format = '$directory$character'
+```
+
+To show the directory and (some) git state information - on a separate line:
+
+```toml
+format = '$directory$git_branch$git_commit$git_state$git_status$line_break$character'
+```
+
+To show the directory and git state information, followed by conditional information about the last command (command duration, status), and shell and prompt character:
+
+```toml
+format = '$directory$git_branch$git_commit$git_state$git_status(- $status$cmd_duration)$line_break$shell$character'
+
+[status]
+# enable status, which is disabled by default
+disabled = false
```

Comment:
I think I would remove status now. On actual use, the exit code is often not useful, and the $character already shows state. The $cmd_duration will still show the concept. *(I removed it in my own config.)*

```suggestion
format = '$directory$git_branch$git_commit$git_state$git_status(- $cmd_duration)$line_break$shell$character'
```